src/locale/locales/el/messages.po

```diff
@@ -0,0 +1,8024 @@
+msgid "<0>{0}</0> and<1> </1><2>{1} </2>are included in your starter pack"
+msgstr "<0>{0}</0> και <1> </1><2>{1} </2>περιλαμβάνονται στο starter pack σας"
+
+#: src/screens/StarterPack/Wizard/index.tsx:509
+msgid "<0>{0}</0> is included in your starter pack"
+msgstr "<0>{0}</0> περιλαμβάνεται στο starter pack σας"
+
+#: src/components/WhoCanReply.tsx:274
+msgid "<0>{0}</0> members"
+msgstr "<0>{0}</0> μέλη"
+
+#: src/components/dms/DateDivider.tsx:69
+msgid "<0>{date}</0> at {time}"
+msgstr "<0>{date}</0> στις {time}"
+
+#: src/screens/Settings/NotificationSettings.tsx:71
+msgid "<0>Experimental:</0> When this preference is enabled, you'll only receive reply and quote notifications from users you follow. We'll continue to add more controls here over time."
+msgstr "<0>Πειραματικό:</0> Όταν αυτή η προτίμηση είναι ενεργοποιημένη, θα λαμβάνετε ειδοποιήσεις απάντησης και παραθέσεων μόνο από χρήστες που ακολουθείτε. Θα συνεχίσουμε να προσθέτουμε περισσότερους ελέγχους εδώ με την πάροδο του χρόνου."
+
+#: src/screens/StarterPack/Wizard/index.tsx:466
+msgid "<0>You</0> and<1> </1><2>{0} </2>are included in your starter pack"
+msgstr "<0>Εσείς</0> και <1> </1><2>{0} </2>περιλαμβάνονται στο starter pack σας"
+
+#: src/screens/Profile/Header/Handle.tsx:52
+msgid "⚠Invalid Handle"
+msgstr "⚠Μη έγκυρο όνομα χρήστη"
+
+#: src/components/dialogs/MutedWords.tsx:193
+msgid "24 hours"
+msgstr "24 ώρες"
+
+#: src/screens/Login/LoginForm.tsx:250
+msgid "2FA Confirmation"
+msgstr "Επιβεβαίωση 2FA"
+
+#: src/components/dialogs/MutedWords.tsx:232
+msgid "30 days"
+msgstr "30 ημέρες"
+
+#: src/components/dialogs/MutedWords.tsx:217
+msgid "7 days"
+msgstr "7 ημέρες"
+
+#: src/Navigation.tsx:361
+#: src/screens/Settings/AboutSettings.tsx:24
+#: src/screens/Settings/Settings.tsx:207
+#: src/screens/Settings/Settings.tsx:210
+msgid "About"
+msgstr "Σχετικά"
+
+#: src/view/com/util/ViewHeader.tsx:89
+#: src/view/screens/Search/Search.tsx:883
+msgid "Access navigation links and settings"
+msgstr "Πρόσβαση σε συνδέσμους πλοήγησης και ρυθμίσεις"
+
+#: src/view/com/home/HomeHeaderLayoutMobile.tsx:56
+msgid "Access profile and other navigation links"
+msgstr "Πρόσβαση στο προφίλ και άλλους συνδέσμους πλοήγησης"
+
+#: src/screens/Settings/AccessibilitySettings.tsx:42
+#: src/screens/Settings/Settings.tsx:183
+#: src/screens/Settings/Settings.tsx:186
+msgid "Accessibility"
+msgstr "Προσβασιμότητα"
+
+#: src/Navigation.tsx:321
+msgid "Accessibility Settings"
+msgstr "Ρυθμίσεις Προσβασιμότητας"
+
+#: src/Navigation.tsx:337
+#: src/screens/Login/LoginForm.tsx:176
+#: src/screens/Settings/AccountSettings.tsx:41
+#: src/screens/Settings/Settings.tsx:145
+#: src/screens/Settings/Settings.tsx:148
+msgid "Account"
+msgstr "Λογαριασμός"
+
+#: src/view/com/profile/ProfileMenu.tsx:132
+msgid "Account blocked"
+msgstr "Λογαριασμός μπλοκαρισμένος"
+
+#: src/view/com/profile/ProfileMenu.tsx:145
+msgid "Account followed"
+msgstr "Λογαριασμός ακολουθούμενος"
```

Comment:
Maybe like this?

Ακολουθούμενος λογαριασμός

Reply:
Ακολουθείται